components/user_settings/advanced/user_settings_advanced.jsx

```diff
@@ -134,6 +144,27 @@ export default class AdvancedSettingsDisplay extends React.Component {
         );
     }
 
+    handleDeactiveSubmit = () => {
+        const userId = UserStore.getCurrentId();
+
+        this.setState({isSaving: true});
+        updateActive(userId, false, null, this.state.serverError);
```

Comment:
The last param is callback function for on error. May need to pass a function to set state of `serverError` whenever error occurred.

Reply:
done! thanks for the explanation